src/Pages/Wishlist-page/wishlist.css

```diff
@@ -0,0 +1,34 @@
+.wishlist-page {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 2.5rem minmax(calc(100vh - 7.5rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+}
+.wishlist__main {
+  grid-area: main;
+  background-color: var(--secondary-bg-color);
+}
+
+/* Wishlist Header */
+.wishlist__header {
+  margin-top: 3rem;
+  text-align: center;
+  color: var(--primary-color);
+  font-size: var(--font-size-large);
+}
+
+/* card Product Ends Here */
+::-webkit-scrollbar {
+  width: 5px;
+  height: 5px;
+}
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 10px rgb(255, 255, 255);
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: var(--primary-color);
```

Comment:
As you are changing browser default style I suggest you to add this style for other browser also at least  for Mozilla

Reply:
> 

Thankyou